concent_api/api-e2e-multi-threads-test.py

```diff
@@ -119,6 +128,32 @@ def send_correct_force_get_task_result(
     responses_global.append(response.__class__.__name__)
 
 
+def send_correct_force_subtask_results(
+    cluster_url: str,
+    signed_report_computed_task: ReportComputedTask,
```

Comment:
why is `signed_report_computed_task` needed?

Reply:
In all tests `report_computed_task` is signed. The naming is confusing, because in all other places there is just `report_computed_task` and here `signed_report_computed_task`. All of them are signed, my mistake for confusing names. Renamed